docs/app/Examples/elements/Input/Variations/InputExampleLeftActionLabeledButton.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Input } from 'semantic-ui-react'
+
+const InputExampleLeftActionLabeledButton = () => (
+  <Input
+    action={{ color: 'teal', label: true, icon: 'cart', content: 'Checkout' }}
```

Comment:
The previous `labeled` prop became `labelPosition`.  This should be `labelPosition: 'left'` which will place the `icon` as a label on the left.